### PR TITLE
native first functionality commit

### PR DESCRIPTION
**Pull request by faraz-2021**

### CODE REVIEW

File: src/components/TodoHeader.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+     paddingRight:10,
+    
+     
+
+    },
+    t1:{
+        fontSize:30,    
+        fontWeight:"bold",
+    },
+});
```
Comment (priyarajputexc) [inline src/components/TodoHeader.js]: 1. format code properly before pushing.
2. add a new empty line in the end of each file
3. create a constants file to store colors.

File: src/components/TodoHeader.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
+import { StatusBar } from 'expo-status-bar';
```
Comment (priyarajputexc) [inline src/components/TodoHeader.js]: Do not import things you don't need.

File: src/components/TodoPopUp.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+import { StatusBar } from 'expo-status-bar';
```
Comment (priyarajputexc) [inline src/components/TodoPopUp.js]: same here

File: src/components/TodoHeader.js (hunk 1 above)
Comment (faraz-2021) [reply src/components/TodoHeader.js]: done with two of these comment

File: src/components/TodoPopUp.js [hunk 4]
```diff
@@ -1,74 +1,105 @@
...
+        flexDirection: "row",
+        justifyContent: "space-between",
+        alignItems: "center",
+
+
+    },
+    innerView:{
+        borderRadius: 20,
+        backgroundColor: "white",
+        shadowColor: "#000",
```
Comment (priyarajputexc) [inline src/components/TodoPopUp.js]: use constant colors. not these strings

File: src/components/color/color.js [hunk 5]
```diff
@@ -3,6 +3,6 @@ export const Colors = {
     red: '#F24444',
     gold: '#D9C57E',
     black:"000000",
-    white:"ffffff",
+    white:"#FFFFFF",
+    blue: "#18BBD5",
   };
```
Comment (priyarajputexc) [inline src/components/color/color.js:5]: remove unused colors

File: src/components/TodoPopUp.js [hunk 6]
```diff
@@ -1,6 +1,7 @@
 import { StatusBar } from 'expo-status-bar';
 import React, { useState } from 'react';
 import { StyleSheet, Text, Touchable, TextInput, TouchableOpacity, View, Modal, TouchableHighlight } from 'react-native';
```
Comment (priyarajputexc) [inline src/components/TodoPopUp.js]: remove unused imports. it increases page load speed

File: App.js [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+      
+    </ScrollView>
+    </>
+    
+  );
+}
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
```
Comment (priyarajputexc) [inline App.js:27]: constant color here

File: src/components/List.js [hunk 8]
```diff
@@ -0,0 +1,58 @@
...
+    );
+  }
+  
+  const styles = StyleSheet.create({
+    container: {
+      flex: 1,
+      marginTop: StatusBar.currentHeight || 0,
+    },
+    item: {
+      backgroundColor: '#f9c2ff',
```
Comment (priyarajputexc) [inline src/components/List.js:47]: constant color here

File: src/components/TodoHeader.js [hunk 9]
```diff
@@ -0,0 +1,56 @@
...
+            </View>
+
+        </View>
+    )
+}
+
+
+const styles = StyleSheet.create({
+    Main: {
+        backgroundColor: '#fff',
```
Comment (priyarajputexc) [inline src/components/TodoHeader.js]: constant color here

File: src/components/TodoPopUp.js [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+    },
+    modalView: {
+        height: 200,
+        width: 300,
+        backgroundColor: Colors.white,
+        borderRadius: 20,
+        padding: 35,
+        alignItems: "center",
+        shadowColor: "#000",
```
Comment (priyarajputexc) [inline src/components/TodoPopUp.js]: constant color here

File: src/components/TodoHeader.js (hunk 2 above)
Comment (faraz-2021) [reply src/components/TodoHeader.js]: ok

**Pull request merged**